target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ board_config_update
 board="$(cat /tmp/sysinfo/board_name)" 2>/dev/null
 
 case "$board" in
+pc-engines-apu2)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	ucidef_set_interface_raw "opt" "eth2"
```

Comment:
@dwmw2 I am fine with this change if it puts the board in line with other LEDE devices. If you don't mind, please give me a short bit to do the testing (30 minutes) and we can decide where to go from there.